src/main/kotlin/org/akuleshov7/stargazers/StargazersCalculator.kt

```diff
@@ -45,13 +43,12 @@ class StargazersCalculator(private val repositories: Set<String>,
         logInfo("Number of stargazers: ${uniqueStargazers.size}")
         logInfo("Number of duplicated stars: $numberOfDuplicatedStars")
 
-        if (extended) {
+        if (config.isExtended) {
             logInfo("Unique stargazers: ${uniqueStargazers.joinToString()}")
-            logInfo("Duplicated stars: ${
-                duplicatedStargazers
-                        .map { "User:${it.key.login} Stars:${it.value}" }
-                        .joinToString("; ")
-                }"
+            logInfo(
+                    "Duplicated stars: ${
+                    duplicatedStargazers.map { "User:${it.key.login} Stars:${it.value}" }.joinToString("; ")
```

Comment:
BTW, shouldn't diktat increase indentation in case of curly braces in string templates?

Reply:
please create an issue for this @petertrr

Reply:
https://github.com/cqfn/diKTat/issues/758